src/scripts.ts

```diff
@@ -0,0 +1,22 @@
+export const projectScripts = () => {
+  return {
+    "manifest": {
+      "script": {
+        "default":
+          "deno run -q --unstable --import-map=import_map.json --allow-read --allow-net https://deno.land/x/deno_slack_builder@0.0.5/mod.ts --manifest",
```

Comment:
nit: Since the `deno_slack_builder` is in multiple places and will need to be updated it'd be nice to just set and use a variable here instead